packages/cli/cli.js

```diff
@@ -144,12 +147,55 @@ yargs.command(
 yargs.command(
     "disable-tracking",
     "Disable tracking of Webiny stats.",
-    () => {},
+    () => {}, // eslint-disable-line
     async () => {
         await require("./sls/disableTracking")();
     }
 );
 
+yargs.command(
+    "scaffold <name>",
+    "Scaffold your application with one of the available templates. (Run it from within your app's folder)",
+    yargs => {
+        yargs.positional("name", {
+            describe: "The name to be assigned to your generated template."
+        });
+    },
+    async argv => {
+        const regexIsCliModule = /(^@webiny\/cli-scaffold-.+$)|(^webiny-cli-scaffold.*$)/;
+        const moduleToChoice = module => `${module.scaffold.name} - ${module.scaffold.description}`;
+
+        const webinyModulesNames = (await fs.readdir("../node_modules/@webiny")).map(
+            moduleName => `@webiny/${moduleName}`
+        );
+        const nodeModulesNames = await fs.readdir("../node_modules");
+        const scaffoldModulesNames = webinyModulesNames
+            .concat(nodeModulesNames)
+            .filter(moduleName => moduleName.match(regexIsCliModule));
+        const scaffoldModules = scaffoldModulesNames.reduce(
+            (modules, crtModuleName) => modules.concat(require(crtModuleName)),
+            []
+        );
```

Comment:
@Fsalker With `fs.readdir("../node_modules/@webiny")` you are limiting the modules to only those located in `{cwd}/../node_modules` which is not very robust. So your code is very dependent on the CWD. Also that's a lot of code for something that can be achieved with a single line and cover all node_modules relevant to your project.

Reply:
Very good point @Pavel910 🚀

Reply:
I would still use the `PluginsContainer` we've mentioned in our conversations here.

Have in mind when we require the modules on line 175, that these modules can export an array of plugins, or a single plugin.

Using `new PluginsContainer(...)` will enable you to "not care" about that fact, and also will enable you later down the road to query registered plugins by type via `pluginsContainer.byType(...)` method, or by name `pluginsContainer.byName(...)`.